atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -131,7 +131,8 @@ def allow_inheritance(self):
         self.log.info("Inheritance requested in config file")
```

Comment:
It seems like we could update this log statement as well to mention the "config file" we're talking about

Reply:
added